app/scripts/directives/ticketFinderContentDriver.dir.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+angular.module('yourStlCourts').directive('ticketFinderContentDriver', [function () {
+  return {
+    restrict: 'E',
+    transclude: false,
```

Comment:
You don't need the `[]` around the function in this file. ng-annotate handles minification protection for us in this project.